### PR TITLE
MB 3484 Pre Demo Cleanup

### PR DESCRIPTION
**Pull request by Shkeating**
## Description

This branch makes the changes requested in this document which includes:
- [x] adding space before the optional label
- [x] spacing on form section headers
- [x] radio button spacing
- [x] *try* switching over everything to Public Sans
- [x] margin and sizing of hhg move page header 

## Reviewer Notes

I am having trouble figuring out where I need to go to switch us off of Source sans. I also could use help figuring out how to navgate to the page where this header is located so I can adjust its styling:
image

## Setup

In storybook:
```sh
yarn install
yarn stroybook
```

In the app
```sh
make server_run
make client_run
```

## Code Review Verification Steps

* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-3484?atlOrigin=eyJpIjoiY2ZmNmVhNTZiMjc3NDcwNWEzNjliZTNjOThhYTc0YjUiLCJwIjoiaiJ9) for this change
* [Google Doc outlining requested changes](https://docs.google.com/document/d/16XtccDWlKDfhC1d5lnHTXmrKgYvth7i9erOxetDvnLs/edit?usp=sharing) explains more about the approach used.

## Screenshots
image
image
image
image

### CODE REVIEW

File: src/shared/styles/_custom.scss [hunk 1]
```diff
@@ -328,3 +333,7 @@ table {
     }
   }
 }
+
+.usa-label:first-of-type {
```
Comment (suzubara) [inline src/shared/styles/_custom.scss:337]: Can this be scoped down at all? Seems like it could cause side effects

**Comment by kimallen** [comment]
The changes look good to me in storybook. Unfortunately, the styles don't show up locally.  Sounds like the reason is related to Suz's comment about the `ghc_index.scss` file not being imported into the customer-facing app.

File: src/shared/styles/_custom.scss (hunk 1 above)
Comment (Shkeating) [reply src/shared/styles/_custom.scss:337]: yup I can do this in a CSS module instead, I forgot that this wouldn't get imported over there!

**Comment by kimallen** [comment]
Re-reviewing, based on local envt:
Spacing between the radios for Delivery address look better
I'm not seeing spacing between optional and the label
I do see better spacing between form section headers

There's changes to buttons that I don't think are intended:
The button is now width of section, don't think it was before
image

and 

The button colors look funny, and on the browser (non-phone view) the buttons have moved directly under the form.  It looks more like a mobile view on the app view.
image

**Pull request merged**